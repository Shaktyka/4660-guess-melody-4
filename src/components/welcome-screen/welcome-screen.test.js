import React from 'react';
import renderer from 'react-test-renderer';
import WelcomeScreen from './welcome-screen.jsx';

const ERRORS_AMOUNT = 4;
const welcomeButtonClickHandler = () => {};

describe(`WelcomeScreen rendering`, () => {

  it(`WelcomeScreen renders correctly`, () => {
    const tree = renderer
      .create()
      .toJSON(
          <WelcomeScreen
            errors={ERRORS_AMOUNT}
            welcomeButtonClickHandler={welcomeButtonClickHandler}
          />
      );

    expect(tree).toMatchSnapshot();
  });

});
