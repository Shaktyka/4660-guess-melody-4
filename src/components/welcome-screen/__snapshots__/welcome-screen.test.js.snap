// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeScreen rendering WelcomeScreen renders correctly 1`] = `null`;
